mep/accounts/tests/test_accounts_admin.py

```diff
@@ -65,3 +65,99 @@ def test_clean(self):
         assert form.is_valid()
         assert form.cleaned_data['end_date'] == form_data['end_date']
 
+class TestBorrowAdminForm(TestCase):
+
+    def test_get_initial_for_field(self):
+        acct = Account.objects.create()
+        acct.save()
+        borrow = Borrow(account=acct)
+        borrow.partial_start_date = '--05-03'
+        borrow.partial_end_date = '1900-05'
+        form = BorrowAdminForm(instance=borrow)
+        # ensure that partial dates are auto-populated correctly
+        assert form.get_initial_for_field(form.fields['partial_start_date'],
+            'partial_start_date') == borrow.partial_start_date
+        assert form.get_initial_for_field(form.fields['partial_end_date'],
+            'partial_end_date') == borrow.partial_end_date
+        # shouldn't affect other fields
+        assert form.get_initial_for_field(form.fields['account'],
+            'account') == borrow.account.id
+
+    def test_validation(self):
+        acct = Account.objects.create()
+        acct.save()
+        borrow = Borrow(account=acct)
+        # valid cases
+        form_data = {
```

Comment:
I would add brief comments before each form data section to make it clear what is being tested and also break up the code so it's obvious which sections go together.